python-lib/sharepoint_client.py

```diff
@@ -110,6 +111,15 @@ def setup_login_details(self, login_details):
             self.sharepoint_root = "Shared Documents"
         logger.info("SharePointClient:sharepoint_root={}".format(self.sharepoint_root))
 
+    def apply_paths_overwrite(self, config):
+        advanced_parameters = config.get("advanced_parameters", False)
+        sharepoint_root_overwrite = config.get("sharepoint_root_overwrite", "").strip("/")
+        sharepoint_site_overwrite = config.get("sharepoint_site_overwrite", "").strip("/")
+        if advanced_parameters and sharepoint_root_overwrite:
+            self.sharepoint_root = sharepoint_root_overwrite
+        if advanced_parameters and sharepoint_site_overwrite:
+            self.sharepoint_site = sharepoint_site_overwrite
```

Comment:
what if someone overwrites the site path without overwriting the dir, or the other way around? In the first case, it is annoying because you won't have the good datas displayed when browsing, but won't have any indication of the current dir used. I understand the logic to let a user overwrite just one of the two fields, but is there a way to indicate the current dir/ current site path in use? Maybe write it as a placeholder in the fields?

Reply:
The UX was modified accordingly in [f893619](https://github.com/dataiku/dss-plugin-sharepoint-online/pull/33/commits/f893619a337d3ef9eb7db28d353ad041f44745f4)
<img width="631" alt="Screenshot 2022-02-14 at 14 32 42" src="https://user-images.githubusercontent.com/3763569/153874046-7eb8f432-071c-4e3d-8916-c7daaabf031e.png">
<img width="244" alt="Screenshot 2022-02-14 at 14 31 51" src="https://user-images.githubusercontent.com/3763569/153874063-4f0ecfd3-3dc7-4aa1-b8e6-0e04c390d4dd.png">